File: mobile-native-android/backend-api/src/main/java/com/edham/logistics/dto/LoginResponse.java
package com.edham.logistics.dto;

import lombok.Builder;
import lombok.Data;
import java.util.List;

@Data
@Builder
public class LoginResponse {
    private String token;
    private String type = "Bearer";
    private String refreshToken;
    private Long id;
    private String email;
    private String name;
    private List<String> roles;
}

File: mobile-native-android/app/src/main/java/com/edham/logistics/data/remote/dto/LoginResponse.kt
package com.edham.logistics.data.remote.dto

data class LoginResponse(
    val success: Boolean,
    val token: String?,
    val refreshToken: String?,
    val user: UserDto?,
    val message: String?
)

data class UserDto(
    val id: String,
    val email: String,
    val name: String,
    val role: String,
    val phone: String?
)
